### DISCUSSION TITLE
Change Width of Image ToolTip

### DISCUSSION THREAD

**Discussion by nbeligh** [ng-matero/extensions#217] [Q&A] [answered]
Hello, hope you are fine, currently i'm trying to change the width of image diplayed but it does not work.
Am I missign something ? in the exemple i juste changed another image from internent larger than Angular logo, but it shows only the first 200px of it, yet the height can be changed.
Thank you in advance for you reply :)

**Comment by seb0077** [comment]
same problem here.
seems the tooltip does not want to resize wider than 200px ...

**Reply by nbeligh** [reply, marked as answer]
Hi Seb, 

we put this in our global styles.scss  and it worked fine : 

`.mtx-tooltip{
  max-width:800px!important;
  min-width:400px!important;
}`

**Comment by seb0077** [comment]
thank you, i will try that!
